lib/strings.nix

```diff
@@ -126,6 +126,20 @@ rec {
     # List of input strings
     list: concatStringsSep sep (lib.imap1 f list);
 
+  /* Generate a string by repeating a given string a given number of times.
+
+     Type: repeat :: string -> int -> string
+
+     Example:
+       repeat "A" 3
+       => "AAA"
+  */
+  repeat =
+    # String to repeat
+    s:
+    # Number of times to repeat it
+    n: concatMapStrings (lib.const s) (lib.range 0 (n - 1));
```

Comment:
The argument order is wrong, the integer should come first and the string last, otherwise it does not play nice with currying.

```
repeat 4 "A"
```

Reply:
Already changed that I think?

Reply:
Huh, I don’t know why github showed me an outdated version of this.